UPGRADING.md

```diff
@@ -6,25 +6,26 @@ This guide provides instructions for upgrading to specific versions of Cosmos SD
 
 ### Migration to CometBFT (Part 2)
```

Comment:
nit: can we combine the two migration to comet bft sections into one?

Reply:
Part 2 is unreleased and Part 1 is under v0.47, so I don't think so?

Reply:
oh i thought what is on main is also used in 0.47 so it seemed to be overlapping. 

Oh these are items we changed in our repo. Kind of confusing

Reply:
I don't really follow. There were some thing repetitive from the other PR, but I've removed it here.
There is indeed some category that are the same between unreleased and v0.47, but this is because we've changed more things after v0.47 had its scope fixed. So this is why it is under unreleased.

Reply:
it was mainly confusing to see part 1 and part 2 but they are not in the same release and have two different scopes.

Reply:
We can reword. As it was both related I thought it was clear. I am not sure how to name the "Part 2" otherwise, as I feel like it's still clearly related.